### PR TITLE
Replaced /main/ from /master/

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
Signed-off-by: Dishank Tiwari <dtiwari@progress.com>

### Description

Update /master/ to /main/ across the documentation.

### Check List

- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Review by PrajaktaPurohit — APPROVED** [review]

File: docs-chef-io/content/server/config_rb_backend.md [hunk 1]
```diff
@@ -362,7 +362,7 @@ See [Elasticsearch's configuration documentation](https://www.elastic.co/guide/e
 
 `leaderl.etcd_pool.ibrowse_options`
 : Internal options to affect how requests to etcd are made (see
-  <https://github.com/cmullaparthi/ibrowse/blob/master/doc/ibrowse.html>).
+  <https://github.com/cmullaparthi/ibrowse/blob/main/doc/ibrowse.html>).
```
Comment (kagarmoe) [inline docs-chef-io/content/server/config_rb_backend.md:365]: please confirm

File: docs-chef-io/content/server/config_rb_backend.md [hunk 2]
```diff
@@ -410,7 +410,7 @@ See [Elasticsearch's configuration documentation](https://www.elastic.co/guide/e
 
 ### Chef HA Backend Leader Connection Pool Settings
 
-See <https://github.com/seth/pooler/blob/master/README.org> for details.
+See <https://github.com/seth/pooler/blob/main/README.org> for details.
```
Comment (kagarmoe) [inline docs-chef-io/content/server/config_rb_backend.md:413]: please confirm

**Review by kagarmoe — CHANGES_REQUESTED** [review]
Two changes to confirm

**Review by kagarmoe — APPROVED** [review]
tenor-21141053

**Pull request merged**